### PR TITLE
Frontend with React for two app hello world project

### PR DESCRIPTION
**Pull request by prius2055**
### Frontend with React for two app hello world project

_In this project, I implemented the following;_

- [ ] Created a new React app
- [ ] Created the App component with react-router.
- [ ] Created the Greeting component that will display a greeting. 
- [ ] Created a store, an action, and a reducer that will connect to the API endpoint from the backend to get the random greeting.
- [ ] Displayed the random greeting in my Greeting component.

_The PR link for the Backend is below;_

https://github.com/prius2055/hello-world-backend/pull/1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,169 @@
...
\ No newline at end of file
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
```
Comment (DammyShittu) [inline README.md]: - [ ] Remove this header link if you have no live demo link and section.

**Review by DammyShittu — COMMENTED** [review]
Hi @prius2055,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### Kindly fix the linter errors and the change below

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md (hunk 1 above)
Comment (prius2055) [reply README.md]: LOL

**Review by frank1738 — APPROVED** [review]
Hi @prius2055 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Gitflow
✅ No linter error
✅ frontend is setup correctly 👍 

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**